cal-adapt/climakitae
# EPW format for TMY file export refinement

This PR refines the .epw file format for TMY data as requested by one of our stakeholders.
- Datetime column split into individual columns
- Order of vars revised
- Additional vars included, with their corresponding missing data codes

EPW source info here: https://energyplus.net/assets/nrel_custom/pdfs/pdfs_v23.1.0/AuxiliaryPrograms.pdf 
- Sections 2.2, 2.9, and 2.13 are most relevant for us
- There are some modifications to the header lines and data that are fothcoming but not completed yet. Trello card with these to do items: https://trello.com/c/Dpo9IpmW/450-epw-files

---
**To test**
Run the AMY_to_TMY notebook and produce a EPW file in the last step. Check that it conforms to .epw standards.
